Project 1 (9.12)/bmi calculator/calculator.js

```diff
@@ -0,0 +1,21 @@
+const kgValue = document.querySelector("#weight");
+const cmValue = document.querySelector("#height");
+const calculateBMI = document.querySelector(".btn");
+let resultofBMI = document.querySelector("#results");
+
+
+const alerts =() => {
+    if (kgValue.value == 0 || cmValue.value == 0){
+      resultofBMI.innerHTML = "Please enter values";
+    } else if (kgValue.value < 0 || cmValue.value < 0)
+    resultofBMI.innerHTML = "Negative alers are not allowed";   
+}
+
+const calculateBMIbutton = () => {
+
+  let yourBMI = kgValue.value / Math.pow(cmValue.value/100, 2);
+  resultofBMI.innerHTML = " Twój wynik to " + yourBMI.toFixed(1);
```

Comment:
jak już raz po angielsku, to do końca po angielsku :)

Reply:
Ach wydawalo mi się , ze w zadaniu byla napisana  ta fraza i poprostu mielismy jej uzyc